packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/index.js

```diff
@@ -134,7 +134,7 @@ const ApiTokenListView = () => {
         {!canRead && <NoPermissions />}
         {shouldDisplayDynamicTable && (
           <DynamicTable
-            headers={tableHeaders}
+            headers={getTableHeaders(formatMessage)}
```

Comment:
I'm not a fan of passing `formatMessage()` down as argument. What do you think about having `getTableHeaders` to return only a plain object, which is then formatted by `formatMessage` here. E.g.:

```js
const headers = tableHeaders
  .map((header) => ({
    ...header,
    metadatas: {
      ...header.metadatas,
      label: formatMessage(header.metadatas.label)
   }
}));

// ...

<DynamicTable headers={headers} />
```

I think it would be better because `tableHeaders` is basically a constant (should probably be called `TABLE_HEADERS` (imo)) and not a useful utility function.